Rubrik/Public/New-RubrikLDAPSettings.ps1

```diff
@@ -0,0 +1,94 @@
+﻿#Requires -Version 3
+function New-RubrikLDAPSettings
+{
+  <#  
+      .SYNOPSIS
+      Connects to Rubrik and sets Rubrik cluster settings
+            
+      .DESCRIPTION
+      The New-RubrikSettings cmdlet will set the cluster settings on the system. This does require authentication.
+            
+      .NOTES
+      Adapted by Adam Shuttleworth from scripts by Chris Wahl for community usage
+            
+      .LINK
+      https://github.com/rubrikinc/PowerShell-Module
+            
+      .EXAMPLE
+      New-RubrikSettings -Name "Test LDAP Settings" -baseDN "DC=domain,DC=local" -authServers "192.168.1.8"
+      This will create LDAP settings on the Rubrik cluster defined by Connect-Rubrik function
+  #>
+
+  [CmdletBinding(SupportsShouldProcess = $true,ConfirmImpact = 'High')]
+  Param(
+    # Rubrik server IP or FQDN
+    [String]$Server = $global:RubrikConnection.server,
+    # ID of the Rubrik cluster or me for self
+    [String]$id = '',
+    # API version
+    [ValidateNotNullorEmpty()]
+    [String]$api = $global:RubrikConnection.api,
+    [Parameter(Mandatory=$True)]
+    [string]$name,
+    [Parameter(Mandatory=$True)]
+    [string]$dynamicDNSName,
+    [string]$baseDN,
+    [string[]]$authServers
```

Comment:
Is [string[]] required?

Reply:
In my testing, a multi-value parameter was needed to correctly build the api all body.  Either way there seems to be an unknown error in the api endpoint.  I receive a strange error when trying to add the new LDAP server entry even through the web api page.

Reply:
Understood. I cleaned up the names a bit. I'd like to dig into the body vars to see what is breaking in the private function, but have left the on-off code in there for now since it won't break using the function in the future.

Reply:
I've modified this to the `[array]` param type instead.